frontend/workflows/projectCatalog/src/index.tsx

```diff
@@ -55,11 +55,17 @@ const register = (): WorkflowConfiguration => {
         featureFlag: "projectCatalog",
       },
       config: {
-        path: "/:projectId/config/:configType",
+        path: "/:projectId/config",
         description: "Service Detail Configuration",
         component: Config,
         featureFlag: "projectCatalog",
       },
+      configType: {
```

Comment:
super nit: if we update the above to be `configLanding` we can probably update this to be `configPage`

Reply:
Will do